package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "web-team-tv",
+  "version": "1.0.0",
+  "description": "This is the package for the web team TV",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack --progress",
+    "lint": "eslint src/**/*.{js,jsx} *.js; exit 0",
+    "release": "webpack --progress --optimize-minimize",
+    "serve": "webpack-dev-server --port $PORT",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperoslo/web-team-tv.git"
+  },
```

Comment:
This entire config can be [simplified]:

```json
"repository": "hyperoslo/web-team-tv"
```

[simplified]: https://docs.npmjs.com/files/package.json#repository